static/app/views/issueList/groupSearchViewTabs/draggableTabMenuButton.tsx

```diff
@@ -33,12 +37,49 @@ export function DraggableTabMenuButton({
         }}
         items={menuOptions}
         offset={[-10, 5]}
+        menuFooter={<FeedbackFooter />}
         usePortal
       />
     </TriggerIconWrap>
   );
 }
 
+function FeedbackFooter() {
+  const openForm = useFeedbackForm();
+
+  if (!openForm) {
+    return null;
+  }
+
+  return (
+    <SectionedOverlayFooter>
+      <Button
+        size="xs"
+        icon={<IconMegaphone />}
+        onClick={() =>
+          openForm({
+            messagePlaceholder: t('How can we make custom views better for you?'),
+            tags: {
+              source: 'custom_views',
```

Comment:
We've been using this convention for the recent feedback buttons, can you use that here too?

```
['feedback.source']: '...',
['feedback.owner']: 'issues',
```

Reply:
yup!